routes/index.js

```diff
@@ -0,0 +1,41 @@
+const express = require("express")
+const router = express.Router()
+const session = require('express-session')
+const MongoStore = require('connect-mongo')
+const authRoutes = require('./auth-routes')
+const auth = require('../middleware/authenticate')
+const UserModel = require("../models/user")
+
+const sessionConfig = {
+  name: 'UID', // name of cookie
+  secret: process.env.COOKIE_SECRET, // secret that makes the cookie effective
+  cookie: {
+    maxAge: 1000 * 60 * 60, // time span of cookie in ms
+    secure: false, // set to true in production for HTTPS only access
+    httpOnly: true // true = no access from js
```

Comment:
We might need a way to make a request to see if a user is signed in because the cookie is HTTP only so the front end can't detect if it is there or not. The code on the front end can't see the cookie, the browser will automatically send the cookie back which can be used to validate that the user is logged in. The best thing would be if it was a route to GET current user details. So when the browser refreshes the front end can automatically make a request to that route and either get the user details or an error.

Reply:
So did you want me to just make a route to get the current use? Cause that info is sent back when the user logs-in/out. It's stored in the cookie and can be accessed at anytime(as long as it hasn't expired) by the api with req.session. Is this more of a 'thinking ahead to show the users page/cart?' kind thing?